### PR TITLE
State prerequisites in README.md explicitly

### PR DESCRIPTION
**Pull request by aliok**
State prerequisites in README.md explicitly.

Comments inline.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,17 @@ EOF
 Please refer to [Building the Operator Image](#building-the-operator-image) to
 build your own image.
 
+## Prerequisites
+
+* [`ko`](https://github.com/google/ko)
+
+  Install `ko` with the following command, if it is not available on your machine:
+
```
Comment (mattmoor) [inline README.md]: Remove trailing whitespace:
```suggestion

```

File: README.md [hunk 2]
```diff
@@ -46,6 +46,17 @@ EOF
 Please refer to [Building the Operator Image](#building-the-operator-image) to
 build your own image.
 
+## Prerequisites
```
Comment (aliok) [inline README.md]: This header was referenced above ("...Please make
sure the [prerequisites](#Prerequisites) are installed first...")

File: README.md [hunk 3]
```diff
@@ -58,8 +69,9 @@ or removal of the knative eventing resources.
 The following are all equivalent:
 
 ```
-kubectl get eventings.operator.knative.dev -oyaml
-kubectl get eventing -oyaml
+kubectl get eventings.operator.knative.dev
+kubectl get eventing
+kubectl get ke
 ```
```
Comment (aliok) [inline README.md]: Added `ke` as well.
Got rid of `-oyaml` as it proves no point here.

**Comment by aliok** [comment]
@googlebot I fixed it..

**Comment by aliok** [comment]
/assign @n3wscott

@n3wscott I don't understand why GoogleBot complained :S
I got the CLA signed already, submitted bunch of PR previously.

**Comment by houshengbo** [comment]
@aliok Plz make sure you sign the CLA.

**Comment by aliok** [comment]
@houshengbo I've already signed it.

Let me push force and remove @mattmoor co-authored commit. Maybe there's a problem with his CLA.

**Comment by houshengbo** [comment]
/approve
/lgtm
Thx!!!

File: README.md [hunk 4]
```diff
@@ -46,6 +46,16 @@ EOF
...
+## Prerequisites
+
+* [`ko`](https://github.com/google/ko)
+
+  Install `ko` with the following command, if it is not available on your machine:
+  ```
+  go get -u github.com/google/ko/cmd/ko
+  ```
+
+
```
Comment (houshengbo) [inline README.md]: README.md:58: MD012 Multiple consecutive blank lines

File: README.md [hunk 5]
```diff
@@ -46,6 +46,16 @@ EOF
 Please refer to [Building the Operator Image](#building-the-operator-image) to
 build your own image.
 
+## Prerequisites
+
+* [`ko`](https://github.com/google/ko)
+
+  Install `ko` with the following command, if it is not available on your machine:
+  ```
```
Comment (houshengbo) [inline README.md]: README.md:54: MD031 Fenced code blocks should be surrounded by blank lines

**Comment by houshengbo** [comment]
@aliok Would u minding changing one more time based on the comments from the build job??

File: README.md [hunk 6]
```diff
@@ -46,6 +46,17 @@ EOF
 Please refer to [Building the Operator Image](#building-the-operator-image) to
 build your own image.
 
+## Prerequisites
+
+* [`ko`](https://github.com/google/ko)
+
```
Comment (mattmoor) [inline README.md]: Remove trailing whitespace:
```suggestion

```

File: README.md [hunk 7]
```diff
@@ -46,6 +46,17 @@ EOF
 Please refer to [Building the Operator Image](#building-the-operator-image) to
 build your own image.
 
+## Prerequisites
+
+* [`ko`](https://github.com/google/ko)
+  
+  Install `ko` with the following command, if it is not available on your machine:
+
+
```
Comment (mattmoor) [inline README.md]: Remove trailing whitespace:
```suggestion

```

**Comment by aliok** [comment]
cc @houshengbo fixed the test failures

**Pull request merged**